src/Algebra/Consequences/Base.agda

```diff
@@ -13,14 +13,24 @@ module Algebra.Consequences.Base
 open import Algebra.Core
 open import Algebra.Definitions
 open import Data.Sum.Base
-open import Relation.Binary.Core
+open import Relation.Binary.Consequences
+  using (mono₂⇒monoˡ; mono₂⇒monoʳ)
+open import Relation.Binary.Core using (Rel)
 open import Relation.Binary.Definitions using (Reflexive)
 
-module _ {ℓ} {_•_ : Op₂ A} (_≈_ : Rel A ℓ) where
+module _ {ℓ} {_∙_ : Op₂ A} (_≈_ : Rel A ℓ) where
 
-  sel⇒idem : Selective _≈_ _•_ → Idempotent _≈_ _•_
+  sel⇒idem : Selective _≈_ _∙_ → Idempotent _≈_ _∙_
   sel⇒idem sel x = reduce (sel x x)
 
+  module Congruence (cong : Congruent₂ _≈_ _∙_) (refl : Reflexive _≈_) where
+
+    ∙-congˡ : LeftCongruent _≈_ _∙_
+    ∙-congˡ = mono₂⇒monoˡ {≤₁ = _≈_} {≤₂ = _≈_} {≤₃ = _≈_} refl cong _
+
+    ∙-congʳ : RightCongruent _≈_ _∙_
+    ∙-congʳ = mono₂⇒monoʳ {≤₁ = _≈_} {≤₂ = _≈_} {≤₃ = _≈_} refl cong _
```

Comment:
Why were the old proofs in `Algebra.Structures` so simple and these are now so complicated?

Reply:
Sigh... because of my DRY-inspired insistence on having `Congruent` delegate to `Monotonic`... and I might need to row back on that design decision. Oh dear...